wgpu-types/src/lib.rs

```diff
@@ -0,0 +1,506 @@
+use std::{io, slice};
+
+#[repr(u8)]
+#[derive(Clone, Copy, Debug, PartialEq)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+pub enum Backend {
```

Comment:
This one doesn't sounds like you'd want from the Web side?

Reply:
Yeah I debated whether to include this here. It's part of the public API wgpu-rs, so it's slightly easier to include another `BrowserWebGpu` variant (possibly even `BrowserWebGl` later) instead of trying to redefine `Backend` for the wasm32 target.